app/src/main/res/layout/item_watchlist_language.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Should we use a fixed size for language code view, just like what we do in the search screen?
![Screenshot_20201208-133237_Wikipedia Dev](https://user-images.githubusercontent.com/2435576/101546514-be288380-395d-11eb-9d63-a697f588da72.jpg)

Reply:
I fixed it so that it properly expands horizontally.